src/smart-components/role/RolesTable.tsx

```diff
@@ -94,11 +94,26 @@ const RolesTable: React.FunctionComponent<RolesTableProps> = ({ selectedRole })
     fetchData({
       limit: perPage,
       offset: (page - 1) * perPage,
-      orderBy: `${direction === 'desc' ? '-' : ''}${sortBy}` || '',
+      orderBy: `${direction === 'desc' ? '-' : ''}${sortBy}`,
       count: totalCount || 0,
       filters: filters,
     });
-  }, [fetchData, page, perPage, filters, onSetFilters, sortBy, direction]);
+  }, [fetchData, page, perPage, sortBy, direction]);
+
+  useEffect(() => {
+    debouncedFetch(
+      () =>
+        fetchData({
+          limit: perPage,
+          offset: (page - 1) * perPage,
+          orderBy: `${direction === 'desc' ? '-' : ''}${sortBy}`,
+          count: totalCount || 0,
```

Comment:
this one can be removed as well

Reply:
sorry what can be removed? The count?

Reply:
yes, it's just a response thing. It doesn't have to be passed in the request